azerothcore/azerothcore-wotlk
# fix (DB): fix Ravager Spicimens overpopulation

## Changes Proposed:
-  This fixes the overpopulation of spawns for Ravager Spicimens

## Issues Addressed:
https://github.com/azerothcore/azerothcore-wotlk/issues/7327
- Closes 

## SOURCE:
Sniff Team

## Tests Performed:

- Build
- Performed


## How to Test the Changes:


1. go c 17199
2. You will see the area is overpopulated, for example 91 on azerothcore there is only 45 in this area on retail and upon checking these npcs are only in this particular area, ofc check and confirm if needed.
